chroma-manager/chroma_core/migrations/0039_add_primary_umtm_stepresults.py

```diff
@@ -0,0 +1,1505 @@
+# -*- coding: utf-8 -*-
+import cPickle as pickle
```

Comment:
Sorry to keep harping on this, but is this import being used anywhere?

Reply:
it is a quicker implementation than the standard pickle module, I'm in the habit of using it